internal/eudev/pkg.yaml

```diff
@@ -0,0 +1,46 @@
+name: eudev
```

Comment:
I feel we need a better story here, I don't want to duplicate every pkg we have in `pkgs/` already. I will think what we can do to provide better story here

Reply:
I guess we can depend on the published image from `pkgs`

Reply:
yep, exactly, I just wanted to to a bit of refactoring to make this easier